peps/pep-0738.rst

```diff
@@ -233,28 +243,33 @@ Python's standard importer.
 Standard library
 ----------------
 
+Unsupported modules
+'''''''''''''''''''
+
 A number of standard library modules will not be supported on Android because
 the underlying C APIs are not available:
 
 * ``curses`` and ``readline``
 * ``dbm.gnu`` and ``dbm.ndbm``
-* ``grp`` and ``spwd``
+* ``grp``
 * ``multiprocessing`` – although subprocesses in general are allowed (see `App
   lifecycle`_), Android does not support any part of the `System V IPC API
   <https://man7.org/linux/man-pages/man7/sysvipc.7.html>`__.
 * ``tkinter`` and ``turtle`` – these would require an Android build of Tk
   itself, which is not officially supported.
 
-
-Platform identification
------------------------
-
 ``sys``
 '''''''
 
 ``sys.platform`` will return ``"android"``. Although Android is based on Linux,
 it differs in enough significant ways that a separate name is justified.
 
+When embedded in an Android app, the C-level stdio streams are not connected to
+anything. So in this mode, ``sys.stdout`` and ``sys.stderr`` will be redirected
+to the system `Logcat <https://developer.android.com/studio/debug/logcat>`__,
+which can be viewed with the Android development tools. ``sys.stdin`` will
+simply return EOF.
```

Comment:
Very minor language style nit - I find it's advisable to avoid the use of "simply", "just", and other words that downplay the complexity of the thing being done. This is especially important in tutorial-like material; but even in technical specification work like this one, the extra word doesn't add any meaning - and in this case, downplays the complexity associated with redirecting stdin at the OS level.

Reply:
Here's a good talk on avoiding "simply": 

* https://www.writethedocs.org/videos/prague/2018/don-t-say-simply-jim-fisher/
* https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/

Reply:
OK, I've replaced it with "always".